docker-compose.yml

```
@@ -0,0 +1,33 @@
+version: '2'
+services:
+  web:
+    build: .
+    restart: always
+    volumes:
+      - .:/var/app
+      - ~/.bash_history:/root/.bash_history
+    ports:
+      - "4000:3000"
+    environment:
+      - "RAILS_ENV=development"
+    depends_on:
+      - postgres
+    networks:
+      - default
+
+  postgres:
+    image: postgres:12.1-alpine
+    environment:
```

had some issues putting up and made some modifications. Not sure if you had something on a .env or something
```suggestion
    container_name: postgres_agendamento_vacina
    environment:
```